src/d3d11/d3d11_query.cpp

```diff
@@ -126,7 +126,7 @@ namespace dxvk {
   
   
   void STDMETHODCALLTYPE D3D11Query::GetDevice(ID3D11Device **ppDevice) {
-    *ppDevice = ref(m_device);
```

Comment:
I don't get this change.

Reply:
ref(m_device) won't compile after m_device is Com<D3D11Device>. Or am I missing something obvious?

Reply:
Nope, I just forgot about that quirk.